WORKSPACE

```diff
@@ -99,10 +93,36 @@ load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
 
 gazelle_dependencies()
 
+load("@io_bazel_rules_docker//go:image.bzl", _go_image_repos = "repositories")
+
+_go_image_repos()
+
 load("//:go_dependencies.bzl", "bb_browser_go_dependencies")
 
 bb_browser_go_dependencies()
 
+load("@com_github_buildbarn_bb_remote_execution//:go_dependencies.bzl", "bb_remote_execution_go_dependencies")
```

Comment:
`go_dependencies.bzl` appears to be missing from https://github.com/buildbarn/bb-remote-execution/pull/28

Reply:
Fixed!